### PR TITLE
Split manage software for pop

### PR DESCRIPTION
**Pull request by ahoneybun**
This will add Flatpak/Flathub instructions and details for Pop 20.04 LTS release.

### CODE REVIEW

File: _articles/manage-repos-pop.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  Add, remove and edit repositories in Pop!_OS
+keywords:
+  - Pop!_OS 18.04
+  - Pop!_OS 19.10
+  - repoman
+  - System76
+  - apt
+image: http://support.system76.com/images/pop-icon.png
+hidden: false
+section: pop-ubuntu
```
Comment (storyaddict) [inline _articles/manage-repos-pop.md:14]: pop not pop-ubuntu

File: _articles/manage-repos-ubuntu.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+description: >
+  Add, remove and edit repositories in Ubuntu
+keywords:
+  - Ubuntu 18.04
+  - Ubuntu 19.10
+  - System76
+  - apt
+image: http://support.system76.com/images/pop-icon.png
+hidden: false
+section: pop-ubuntu
```
Comment (storyaddict) [inline _articles/manage-repos-ubuntu.md:13]: ubuntu not pop-ubuntu

File: _articles/manage-repos-pop.md (hunk 1 above)
Comment (ahoneybun) [reply _articles/manage-repos-pop.md:14]: Fixed here : https://github.com/system76/docs/pull/334

File: _articles/manage-repos-ubuntu.md (hunk 2 above)
Comment (ahoneybun) [reply _articles/manage-repos-ubuntu.md:13]: Fixed here: https://github.com/system76/docs/pull/334

**Pull request merged**